package.json

```diff
@@ -41,9 +41,13 @@
   "jest": {
     "snapshotSerializers": [
       "enzyme-to-json/serializer"
+    ],
+    "transformIgnorePatterns": [
```

Comment:
The `juju-react-components` package doesn't export a commonjs build so this needs to tell Jest to build them. We can remove this when we bring the Terminal component in to the project.